lib/setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.2.1.post1
```

Comment:
do we want to bump to `post2` or just `0.2.2`?

Reply:
Yeah i think a `.post` release should be good here cuz there weren't any code change differences after 0.2.1, just the dependency versions which is why i chose a post